global.php

```diff
@@ -760,6 +760,12 @@
 	eval('$pm_notice = "'.$templates->get('global_pm_alert').'";');
 }
 
+$remote_avatar_notice = '';
+if($mybb->user['avatartype'] === 'remote' && !$mybb->settings['allowremoteavatars'])
```

Comment:
`$mybb->user['avatartype']` equals `gravatar` for that so it doesn't show this warning if you were using a gravatar.